docker/entrypoint.sh

```diff
@@ -4,8 +4,8 @@ cat <<EOF > /etc/orchestrator.conf.json
 {
   "Debug": true,
   "ListenAddress": ":3000",
-  "MySQLTopologyUser": "${ORC_TOPOLOGY_USER:-:orchestrator}",
-  "MySQLTopologyPassword": "${ORC_TOPOLOGY_PASSWORD:-:orchestrator}",
```

Comment:
Oh! You are right, of course. I'm not sure if there was an intention behind `:orchestrator`. Looking at blame/history but I don't find anything that suggests the `:` is of importance. 
Since I'm not familiar with docker, dumb question: is this perhaps a template variable syntax or similar?